java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -100,4 +111,42 @@ protected Object handleObjectResponse(Response response) {
         // convert protobuf response into Object
         return new BaseCommandResponseResolver(RedisValueResolver::valueFromPointer).apply(response);
     }
+
+    /**
+     * Extracts the response value from the Redis response and either throws an exception or returns
+     * the value as a <code>String</code>.
+     *
+     * @param response Redis protobuf message
+     * @return Response as a <code>String</code>
+     * @throws RedisException if there's a type mismatch
+     */
+    protected String handleStringResponse(Response response) {
```

Comment:
You can do
```suggestion
    protected String handleStringResponse(Response response) throws RedisException {
```
without any other code changes, because `RedisException` is unchecked

Reply:
Added